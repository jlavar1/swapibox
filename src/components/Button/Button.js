import React, { Component } from 'react';

class Button extends Component {
  constructor() {
    super();
    this.state = {}
  }

  render() {
    return (
      <div>
        <button>People</button>
        <button>Planets</button>
        <button>Vehicles</button>
      </div>
    )
  }
}

export default Button