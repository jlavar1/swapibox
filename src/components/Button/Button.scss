.category-button {
  background: #feda4a;
  font-family: 'Poller One', cursive;
  border-radius: 2px;
  color: black;
  font-size: 1em;
  margin-top: 5px;
  padding: 5px 10px;
  text-decoration: none;
  &:hover {
    background:darken(#feda4a, 20%);
    cursor: pointer;
    transition: all 0.3s ease;
  }
  &:active {
    background:darken(#feda4a, 40%);
  } 
}

.button-container {
  width: 100%;
  display: flex;
  justify-content: space-around;
  margin-top: 2rem;
}

.Buttons {
  background-color: black;
  border: solid 2px #feda4a
}