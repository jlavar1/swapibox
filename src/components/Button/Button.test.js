import React from 'react';
import Button from './Button';
import { shallow } from 'enzyme';

describe('Button', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(
      <Button />
    );
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

  it('should have the proper default state', () => {
    expect(wrapper.state()).toEqual({});
  });
})