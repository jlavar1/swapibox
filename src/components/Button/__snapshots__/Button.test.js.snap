// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot 1`] = `
<div>
  <button>
    People
  </button>
  <button>
    Planets
  </button>
  <button>
    Vehicles
  </button>
</div>
`;
